src/EditorFeatures/CSharpTest/CommonMemberData.cs

```diff
@@ -0,0 +1,50 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Microsoft.CodeAnalysis.Editor.CSharp.UnitTests;
+
+public static class CommonMemberData
+{
+    public static IEnumerable<object[]> DataTypeDeclarationKeywords
+    {
+        get
+        {
+            yield return new[] { "class" };
```

Comment:
nit: you should be able to use collection expressions here.  Also, you *shoudl* be able to have this all just be `=> [["struct"], ["record"], ...]`.  No need to yield.